src/vs/platform/environment/common/argv.ts

```diff
@@ -91,6 +91,7 @@ export interface NativeParsedArgs {
 	'__enable-file-policy'?: boolean;
 	editSessionId?: string;
 	'locate-shell-integration-path'?: string;
+	'settings-profile'?: string;
```

Comment:
Should it be `--profile`? I think what we derive from it is up to us, but that would make it more first class.

Reply:
It is named after the feature name **Settings Profile**

Reply:
Yeah I get it, but tbh I think just calling it "Profile" would be nicer. In the end that is what the user is after: changing the profile. The impact of doing so is an implementation detail on us. Settings is part of it, but there is more like UI state that is not controlled via settings.

I would still think `--profile` is much nicer and concise to use.

Reply:
✅